Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -69,14 +69,22 @@
         {
             "name": "WebAppKind",
             "type": "pickList",
-            "label": "Web app kind",
+            "label": "App Type",
             "defaultValue": "app",
             "required": false,
-            "visibleRule": "WebAppName = invalidwebappname",
+            "options": {
+                "app": "Web App",
+                "functionapp": "Function App",
+                "logicapp": "Logic App",
+                "api": "API App",
+                "mobileapp": "Mobile App",
+                "applinux": "Linux App"
```

Comment:
Is it possible to split applinux into 2 based on the image source-builtIn or container registry? that might reduce the number of input selections the user has to make in case of linux apps. Please check with @kmkumaran or PM's

Reply:
We can't merge the source-builtIn option with the apptype input here without breaking compat or without revving up the task version. Discussed with @RoopeshNair and the ask is to retain this as is in current version of task. @Ajay-MS - enhance the help t call out the option supports Linux app deployment using packages or containers.